Jenkinsfile

```diff
@@ -108,231 +114,388 @@ pipeline {
 
     stages {
         /************************************************************************
-         * STAGE
-         * -----
-         * Build Zowe CLI Bundle
-         *
-         * TIMEOUT
-         * -------
-         * 10 Minutes
-         *
-         * EXECUTION CONDITIONS
-         * --------------------
-         * - Always
-         *
-         * DECRIPTION
-         * ----------
-         * Gets the latest version of the Zowe CLI and SCS plugin from Zowe
-         * Artifactory. Creates an archive with 'fat' versions of the CLI and Plugin -
-         *  dependencies are bundled.
-         *
-         * OUTPUTS
-         * -------
-         * A Zowe CLI Archive containing Zowe CLI and Zowe CLI Secure Credential Store Plugin
-         ************************************************************************/
-        stage('Create Zowe CLI Bundle') {
-            when {
-                allOf {
-                    expression {
-                        return BRANCH_NAME.equals(MASTER_BRANCH)
-                    }
-                }
-            }
+        * STAGE
+        * -----
+        * Setup
+        *
+        * DESCRIPTION
+        * ----------
+        * Logs in to NPM registry, downloads licenses ZIP, and installs dependencies
+        ************************************************************************/
+        stage('Setup') {
             steps {
-                timeout(time: 10, unit: 'MINUTES') {
-
+                timeout(time: 5, unit: 'MINUTES') {
                     sh "npm set registry https://registry.npmjs.org/"
                     sh "npm set @zowe:registry ${ZOWE_ARTIFACTORY_URL}"
                     withCredentials([usernamePassword(credentialsId: ARTIFACTORY_CREDENTIALS_ID, usernameVariable: 'USERNAME', passwordVariable: 'PASSWORD')]) {
                         // TODO: Consider using tooling like artifactory-download-spec to get license.zip. Post-Infrastructure migration answer.
-                        sh "mkdir -p licenses && cd licenses && curl -fs -o zowe_licenses_full.zip $ZOWE_LICENSE_ZIP_URL"
+                        sh "curl -fs -o zowe_licenses_full.zip $ZOWE_LICENSE_ZIP_URL"

```

Comment:
Do we want to have a .zip file next to the tgzs with `zowe` in it?
Wondering if this may cause problems for someone doing something like:
`zowe plugins install $(ls | grep zowe)`
It's probably fine though 😋

Reply:
`zowe plugins install $(ls | grep zowe)` would match more unwanted files than just the .zip file, it would also match "zowe-cli.tgz".

To avoid this problem, I typically do `zowe plugins install *-for-zowe-cli.tgz` to be as specific as possible with the wildcard 🙂